### PR TITLE
Fix ComparisonFailure on Windows

### PR DESCRIPTION
**Pull request by gquintana**

### CODE REVIEW

**Comment by gquintana** [comment]
I'd like JMXTrans was also built on a Windows CI (Appveyor for example), in order to check that it builds and runs on Windows.

File: jmxtrans-output/jmxtrans-output-gelf/src/test/java/com/googlecode/jmxtrans/model/output/gelf/GelfWriterFactoryTests.java [hunk 1]
```diff
@@ -227,7 +227,7 @@ public void testNonDefaultsWriterFactory() throws ValidationException {
 		assertThat(
 			gelfWriterFactory.getGelfConfiguration().getTlsTrustCertChainFile()
 				.getPath()
-		).isEqualTo("/somefile")
+		).isEqualTo(File.separator + "somefile")
```
Comment (gehel) [inline jmxtrans-output/jmxtrans-output-gelf/src/test/java/com/googlecode/jmxtrans/model/output/gelf/GelfWriterFactoryTests.java]: Even better would probably to compare File object. I wonder if AssertJ has some support to do this more cleanly...

Comment (gquintana) [inline jmxtrans-output/jmxtrans-output-gelf/src/test/java/com/googlecode/jmxtrans/model/output/gelf/GelfWriterFactoryTests.java]: Do you prefer?
```
		assertThat(
			gelfWriterFactory.getGelfConfiguration().getTlsTrustCertChainFile()
		).hasName("somefile")
			.as("Invalid tls trustcertchain file");
```

Comment (gehel) [inline jmxtrans-output/jmxtrans-output-gelf/src/test/java/com/googlecode/jmxtrans/model/output/gelf/GelfWriterFactoryTests.java]: I do find the second version (`hasName()`) slightly nicer. But honestly, either is fine by me.

**Comment by gehel** [comment]
@gquintana feel free to put Appveyor in place. And ping me if you need some help...

**Pull request merged**